src/components/Filters.vue

```diff
@@ -1,64 +0,0 @@
-<template>
-  <div>
-    <div class="filter-species">
-      <h4 class="text-left">Species</h4>
-      <div class="form-check text-left" v-for="specie in species" :key="specie">
-        <input
-          class="form-check-input"
-          type="checkbox"
-          :value="specie"
-          name="filterSpecies"
-          v-model="filterSpecies"
-          @change="onSpecieChange()"
-        />
-        <label class="form-check-label">{{specie}}</label>
-      </div>
-    </div>
-    <div class="filter-gender">
-      <h4 class="text-left">Gender</h4>
-      <div class="form-check text-left" v-for="item in genders" :key="item">
-        <input
-          class="form-check-input"
-          type="checkbox"
-          :value="item"
-          name="filterGender"
-          v-model="filterGender"
-          @change="onChange()"
-        />
-        <label class="form-check-label">{{item}}</label>
```

Comment:
This should have been a Textbox Component. For accessibility to work perfectly, `<input>` should come under `<label>`. Instead of writing `{{item}}`, there would be two spans, one for sr-reader(Accessibility) and other for the normal label and would be shown conditionally.. 

To show normal label- **showLabel** should be true
For the accessibility, **showSrLabel** should be true and **srText** should be a mandatory prop.

Reply:
I get the accessibility part, but how can I use a textbox in place of checkbox as per designs?

Reply:
Yes, not textbox but a checkbox component(input wrapped inside label)